lib/kennel/syncer.rb

```diff
@@ -27,6 +30,14 @@ def plan
         print_plan "Update", @update, :yellow
         print_plan "Delete", @delete, :red
       end
+
+      Plan.new(
+        noop?: noop?,
+        already_ok: @already_ok,
```

Comment:
these seem a bit redundant, maybe a plan class could hold the attrs and have a `noop?` method ?

Reply:
Maybe. In a later refactor, I would suggest.

Reply:
You're not suggesting changing the interface, just the implementation.